Sources/Graphiti/Subscription/SubscribeField.swift

```diff
@@ -450,7 +450,7 @@ public extension SubscriptionField {
     convenience init<ResolveType>(
         _ name: String,
         at function: @escaping SimpleAsyncResolve<SourceEventType, Context, Arguments, ResolveType>,
-        as: FieldType.Type,
+        as _: FieldType.Type,
```

Comment:
Looks like these changes are unrelated?

Reply:
Kinda - turns out they were required by the format job (here's the failing instance): https://github.com/GraphQLSwift/Graphiti/actions/runs/15783062210/job/44493217106

I'm not sure why they didn't fail before - It seems like something that may have changed in swiftformat since we run the same CI command ([old](https://github.com/GraphQLSwift/Graphiti/blob/cedafbcd776e31ebd346164e7497122969a06e1b/.github/workflows/build.yml#L24) and [new](https://github.com/GraphQLSwift/ci/blob/966e597ba5c08183411c4e621ffc97aa1758479c/.github/workflows/lint.yaml#L24))

Reply:
Is that a breaking change, though? Can’t see if it’s a public API. GR kinda sucks on mobile.